INSTALL.md

```diff
@@ -20,22 +20,10 @@
 conda create --name maskrcnn_benchmark
 conda activate maskrcnn_benchmark
 
-# this installs the right pip and dependencies for the fresh python
-conda install ipython
-
-# maskrcnn_benchmark and coco api dependencies
-pip install -r requirements.txt
-
-# follow PyTorch installation in https://pytorch.org/get-started/locally/
-# we give the instructions for CUDA 9.0
-conda install pytorch-nightly cudatoolkit=9.0 -c pytorch
+conda install -c pytorch pytorch torchvision opencv
```

Comment:
Don't we need the instructions for cuda now that we have changed the way it is setup?
Also, is opencv a hard-requirement? I don't think so, except for the demos, right?

Reply:
yes indeed